crates/tx-graph/src/transactions/deposit.rs

```diff
@@ -557,3 +557,50 @@ mod tests {
         );
     }
 }
+
+pub mod prop_tests {
+    use std::str::FromStr;
+
+    use bitcoin::{hashes::sha256d, Amount, OutPoint, Txid, XOnlyPublicKey};
+    use miniscript::Miniscript;
+    use proptest::{prelude::*, prop_compose};
+    use strata_bridge_primitives::operator_table::prop_test_generators::arb_btc_key;
+
+    use super::DepositRequestData;
+
+    prop_compose! {
+        fn arb_txid()(bs in any::<[u8; 32]>()) -> Txid {
+            Txid::from_raw_hash(*sha256d::Hash::from_bytes_ref(&bs))
+        }
+    }
+
+    prop_compose! {
+        fn arb_outpoint()(txid in arb_txid(), vout in 0..100u32) -> OutPoint {
+            OutPoint { txid, vout }
+        }
+    }
```

Comment:
The deposit is always the first vout `0` in the spec.

Reply:
6da7b335e4cd4a0cb226b115578900b0b4d5f777